shop/src/main/java/ro/msg/learning/configuration/ScheduledTasks.java

```diff
@@ -0,0 +1,26 @@
+package ro.msg.learning.configuration;
```

Comment:
It could be moved to another package. It's not a configuration

Reply:
Fixed